src/styles/blocks/header.scss

```diff
@@ -0,0 +1,55 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-link {
```

Comment:
The '.logo-link' class has 'display: flex;' property but it doesn't seem to use any flexbox features like 'align-items' or 'justify-content'. If it's not necessary, you should remove 'display: flex;' to keep the CSS clean.